WhatToEat/App.xaml.cs

```diff
@@ -9,6 +9,9 @@ public App()
     {
 			InitializeComponent();
         DependencyService.Register<MockDataStore>();
+#if IOS
+        DependencyService.Register<IAsyncAnnouncement, SemanticScreenReaderAsyncImplementation>();
```

Comment:
Can you implement this using ServiceProvider from the build vs DependencyService?

Can you get rid of all references to DependencyService

Reply:
Addressed!